doc.go

```diff
@@ -0,0 +1,4 @@
+/*
+Package websspi provides middleware to require with Windows Integrated Authentication.
```

Comment:
May be we should extend a bit the comment in the doc.go file. Something like that should do:
```suggestion
Package websspi implements a middleware for SSO Kerberos authentication of HTTP requests in Windows environments by SSPI, without the need for any keytab files
```